.gitignore

```
@@ -9,6 +9,8 @@ artifacts/contracts/**/**/*.dbg.json
 .openzeppelin/unknown-298.json
 .env
 test-results.*
+contracts-abi/*
+!contracts-abi/contracts
```

small nit, does it make sense to exclude OpenZeppelin's ABI directly?

```suggestion
contracts-abi/@openzeppelin
```

---

I think the main purpose is to showcase the ABIs of contracts in our repo. So I think leaving OZ's ABI out won't really be a big deal.